pkg/util/defaults.go

```diff
@@ -1,13 +1,17 @@
 package util
 
-func GetIntOrDefault(ptr *int, defaultValue int) int {
+import (
+	"time"
+)
+
+func GetBoolOrDefault(ptr *bool, defaultValue bool) bool {
 	if ptr == nil {
 		return defaultValue
 	}
 	return *ptr
 }
 
-func GetBoolOrDefault(ptr *bool, defaultValue bool) bool {
+func GetTimeDefault(ptr *time.Duration, defaultValue time.Duration) time.Duration {
```

Comment:
Note that this package provides the same functions as the ones defined here:
https://github.com/kubernetes/utils/blob/master/pointer/pointer.go

`DurationDeref`